doc/man/pgmoneta-walfilter.1.rst

```diff
@@ -0,0 +1,102 @@
+=====================
+pgmoneta-walfilter
+=====================
+
+-------------------------------------------------
+Command line utility to filter and process Write-Ahead Log (WAL) files
+-------------------------------------------------
+
+SYNOPSIS
+========
+
+**pgmoneta-walfilter** *yaml_config_file*
+
+DESCRIPTION
+===========
+
+**pgmoneta-walfilter** is a command-line utility that reads and filters PostgreSQL Write-Ahead Log (WAL) files based on user-defined criteria. It processes WAL files from a source directory, applies filtering rules, and outputs the filtered WAL files to a target directory.
```

Comment:
Same

Reply:
In 0.20 it is the parse, list of records, write records and CRC that is functionality